code/upaya/app/assets/stylesheets/application.css.scss

```diff
@@ -11,6 +11,6 @@
  * file per style scope.
  *
  *= require_self
- *= require ./main
+ *= require us_web_design_standards
+ *= require us_web_design_standards_fonts
  */
```

Comment:
Correction: it was partially right before. `main.scss` can import the standards as it did before, but instead of `*= require ./main` in application.css.scss, it should be `@import 'main';`